Source/utils/paths.cpp

```diff
@@ -103,6 +108,8 @@ const std::string &ConfigPath()
 		configPath = FromSDL(IOSGetPrefPath());
 #elif defined(NXDK)
 		configPath = NxdkGetPrefPath();
+#elif defined(__amigaos__)
+		prefPath = std::string("PROGDIR:");		

```

Comment:
Since you made SDL_GetPrefPath return this value is this overwrite really needed?

Reply:
Right, haven't noticed SDL_GetPrefPath there.